cfme/networks/floating_ips.py

```diff
@@ -72,9 +72,14 @@ class Details(CFMENavigateStep):
 
     def step(self):
         # as for 5.9 floating ip doesn't have name att, will get id for navigation
+        # as for 5.8 floating ip doesn't found in table with name att, only with address
```

Comment:
Can we just improve this comment, I think you are trying to say: "for 5.8, if floating ip isn't found using name, we will use address" ? But I am not 100% sure :)